packages/runtime/src/components/builtin/Text/EditableText.tsx

```diff
@@ -171,7 +174,18 @@ const EditableText = forwardRef(function EditableText(
     if (event.relatedTarget == null) editor.select(selection)
   }, [])
 
-  const isInBuilder = useIsInBuilder()
+  // HACK: Slate holds on to the very first DOM event handlers passed in and doesn't update them
+  // even if they change.
+  const lastEditMode = useRef(editMode)
+  if (lastEditMode.current !== editMode) lastEditMode.current = editMode
+
+  const handleClick = (event: ReactMouseEvent, _editor: Editor, next: () => any) => {
+    next()
+
+    // This is needed to prevent clicks from propagating in content mode
+    // https://linear.app/makeswift/issue/PRD-525/need-to-prevent-clicks-from-propagating-in-content-mode
```

Comment:
For an OSS repo I think we should try and keep comments to things that are publicly accessible in case somebody wants to contribute